README.md

```diff
@@ -2,67 +2,31 @@
 An interactive dashboard for users to calculate their individualized risk score of contracting COVID-19 and related adverse health outcomes.
 
 ## Contents
-* app.R
-    * Launch the 19andMe app (https://19andme.covid19.mathematica.org)
-* data/
-    * Store the input data
-* doc/
-    * Calculate input from the literature
-* src/
+
+* app/
     * R Scripts for different modules of the app
     * Google Analytics plug-in
     * CSS
-* www/
-    * Mathematica logo
+    * www/
+        * Mathematica logo
+
 ## Authors
-* **Cindy Hu** - *back-end lead*
-* **George Luo** - *front-end lead*
-* **Emma Pendl-Robinson, Erin Lipman** - *back-end developer*
+* **Cindy Hu** - *product owner*
+* **Emma Pendl-Robinson** - *front-end lead* 
+* **Erin Lipman** - *back-end lead*
+* **Jennifer Starling** - *validation lead*
+* **Margaret Luo** - *data engineering lead*
+* **Max Dulieu** - *API lead*
 
 ## Reviewers and Support
-* **Margaret Luo** - *Docker guru*
+* **George Luo** - *front-end developer*
 * **Arpan Bhattacharya, George Gallo** - *AWS architects*
-* **Kelsey Skvoretz** - *QA reviewer*
-* **Fei Xing** - *QA reviewer*
-* **Matt Salganik** - *Expert reviewer*
+* **Sean Kirk, Aaron White** - *data pipeline guru*

```

Comment:
Adding @margaretluo 's comment about the word _guru_ here: 

> I personally don't really like the use of the word guru in this context because it feels like appropriation from Buddhist and Hindu religions. Could we use the word "expert" or "implementer" instead?